extras/homeassistant/custom_components/open_filament_sensor/config_flow.py

```
@@ -0,0 +1,80 @@
+
+async def validate_input(hass, data: dict[str, Any]) -> dict[str, Any]:
+    """Validate the user input allows us to connect."""
+    session = async_get_clientsession(hass)
+    host = data[CONF_HOST]
+    url = f"http://{host}/sensor_status"
+
+    try:
+        async with asyncio.timeout(10):
+            async with session.get(url) as response:
+                if response.status != 200:
+                    raise CannotConnect(f"HTTP {response.status}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)